src/utils/Recorder.ts

```diff
@@ -0,0 +1,83 @@
+import { MockedRequest } from './handlers/requestHandler'
+import { fetch } from '../context/fetch'
+
+async function createFunctionFromRequest(
+  request: MockedRequest,
+  response: Response,
+) {
+  const lines = []
+  const composeLines = []
+  lines.push(
+    `rest.${request.method.toLowerCase()}('${
```

Comment:
We can also check if the request body/query parameters have `operation`/`variables` properties, treating such request as a GraphQL request.

Reply:
yes, this is a draft. Of course I would support GraphQL